### PR TITLE
Added implementation of Spring Jpa Task

### PR DESCRIPTION
**Pull request by MinoDentori**
1. Configured liquibase, created changelog master, and created yaml file for the "book" table creation.
2. Updated entity to implement the "soft-delete" concept.
3. Added new endpoints: updateBookByid, and deleteBookById to controller and service layer implementations for them.
4. Testes the app via Postman.

### CODE REVIEW

**Review by Andriy-Hrynevetskyi — APPROVED** [review]
GJ!

File: src/main/java/mate/academy/onlinebookstore/service/impl/BookServiceImpl.java [hunk 1]
```diff
@@ -39,4 +40,20 @@ public BookDto findById(Long id) {
         return bookMapper.toDto(foundedBook);
     }
 
+    @Override
+    @Transactional
+    public BookDto updateBookById(Long id, CreateBookRequestDto requestDto) {
+        Book bookFromDb = bookRepository.findById(id).orElseThrow(() ->
+                new EntityNotFoundException("Can't get employee from DB by id:" + id));
```
Comment (d1sam) [inline src/main/java/mate/academy/onlinebookstore/service/impl/BookServiceImpl.java]:
```suggestion
                new EntityNotFoundException("Can't get book from DB by id:" + id));
```

**Review by d1sam — CHANGES_REQUESTED** [review]
Well done, but please be careful with exception messages :)

File: src/main/java/mate/academy/onlinebookstore/service/impl/BookServiceImpl.java (hunk 1 above)
Comment (MinoDentori) [reply src/main/java/mate/academy/onlinebookstore/service/impl/BookServiceImpl.java]: my bad. fixed all

**Review by d1sam — APPROVED** [review]
Well done!

**Pull request merged**